tools/2d_auto_threshold/auto_threshold.xml

```diff
@@ -1,5 +1,12 @@
-<tool id="ip_threshold" name="Auto Threshold" version="0.0.5" profile="20.05">
-   <description>applies a standard thresholding algorithm to an image</description>
+<tool id="ip_threshold" name="Threshold image" version="0.0.5-2" profile="20.05">
+   <description>scikit-image</description>

```

Comment:
why is scikit-image here mentioned?

Reply:
all the tool does is switching between different `threshold_*` functions from the `skimage.filters` module.

Reply:
ok